samples/ImageList/src/views/ImageList.tsx

```diff
@@ -26,7 +26,7 @@ const _styles = {
         backgroundColor: '#fff',
         flexDirection: 'column',
         alignSelf: 'stretch',
-        margin: 0
+        margin: 0,
```

Comment:
In the coding style that we use consistently throughout the ReactXP source base, we don't allow dangling commas. Please remove.